### PR TITLE
Update names to match new naming conventions

### PR DESCRIPTION
**Pull request by cwbollinger**

### CODE REVIEW

File: data/waypoints/1.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+data/waypoints/1: [
```
Comment (iamtesch) [inline data/waypoints/1.txt]: Accidentally added waypoints

File: launch/arm_moveit_node.launch [hunk 2]
```diff
@@ -10,15 +10,15 @@
...
+  (defaults to the HEBI 6-DoF arm A-2085-06).  This should
   match a moveit config that is visible as a package with
   name `hebi_<arm_type>_moveit_config`, as well as a config
   parameters file located at `config/<arm_type>_params.yaml`.
 
   -->
 
   <!-- specify type of HEBI arm -->
-  <arg name="arm_type" default="a-2085-06" />
+  <arg name="arm_type" default="A-2085-06" />
```
Comment (iamtesch) [inline launch/arm_moveit_node.launch]: check that uppercase now matches param files?

File: src/kits/rosie/teach_repeat_controller_node.cpp [hunk 3]
```diff
@@ -17,8 +17,16 @@ int main(int argc, char ** argv) {
...
+  if (node.hasParam("family") && node.getParam("family", family)) {
+    ROS_INFO("Found and successfully read 'family' parameter");
+  } else {
+    ROS_INFO("Could not find/read 'family' parameter; defaulting to 'Rosie'");
+    family = "Rosie";
+  }
+
   // Create mobile IO connection
-  auto io = hebi::MobileIO::create("HEBI", "Mobile IO");
+  auto io = hebi::MobileIO::create(family, "mobileIO");
```
Comment (iamtesch) [inline src/kits/rosie/teach_repeat_controller_node.cpp]: Need to update README.md in this directory to correspond with this.

File: launch/rosie_teach_repeat.launch [hunk 4]
```diff
@@ -25,6 +25,8 @@
     <remap from="/save_waypoint" to="/arm/save_waypoint"/>
 
     <remap from="/gripper_strength" to="/gripper/gripper_strength"/>
+
+    <param name="family" value="Rosie"/>
```
Comment (iamtesch) [inline launch/rosie_teach_repeat.launch]: I think we need to set this family parameter for the `base`, `arm`, and `gripper` nodes above, right?

File: data/waypoints/1.txt (hunk 1 above)
Comment (cwbollinger) [reply data/waypoints/1.txt]: Removed

File: launch/rosie_teach_repeat.launch (hunk 4 above)
Comment (cwbollinger) [reply launch/rosie_teach_repeat.launch]: arm/families and gripper/family are set in the rosparam block, but the base family name does need to be set now that we changed the default from Rosie

**Pull request merged**